### PR TITLE
fix: typo `Paramters` -> `Parameters`

### PR DESCRIPTION
**Pull request by deining**
This PR fixes a typo i spotted.

### CODE REVIEW

File: docs/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.vitepress
```
Comment (CaiJimmy) [inline docs/.gitignore:1]: https://vitepress.dev/guide/getting-started

According to the official documentation we should add `.vitepress/cache` and `.vitepress/dist`. There are useful files inside `.vitepress` that are being excluded by this.

image

But anyway, this PR is a typo fix, so please exclude this change from it.

**Review by CaiJimmy — CHANGES_REQUESTED** [review]
Thanks for the pull request!

File: docs/.gitignore (hunk 1 above)
Comment (deining) [reply docs/.gitignore]: > But anyway, this PR is a typo fix, so please exclude this change from it.

Done.

**Review by CaiJimmy — APPROVED** [review]
Thanks!

**Pull request merged**